frontend/packages/dev-console/src/components/import/deploy-image/ResultsDisplay.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import { Timestamp, units } from '@console/internal/components/utils';
+import { Alert } from '@patternfly/react-core';
+import { useFormikContext, FormikValues } from 'formik';
+
+const runsAsRoot = (image) => {
+  const user = _.get(image, 'dockerImageMetadata.Config.User');
+  return !user || user === '0' || user === 'root';
+};
+
+const ResultsDisplay: React.FC = () => {
+  const { values } = useFormikContext<FormikValues>();
+
+  const ImagePorts = ({ ports }) => (
+    <React.Fragment>
+      {_.size(ports) > 1 ? 'Ports ' : 'Port '}
+      {_.map(ports, (port) => `${port.containerPort}/${port.protocol.toUpperCase()}`).join(
+        ', ',
+      )}{' '}
+      will be load balanced by Service <strong>{values.name || '<name>'}</strong>.
+      <div>
+        Other containers can access this service through the hostname{' '}
+        <strong>{values.name || '<name>'}</strong>.
+      </div>
+    </React.Fragment>
+  );
+
+  return (
+    <div>
+      {!_.isEmpty(values.isi.image) && (
+        <div className="co-image-name-results">
+          <React.Fragment>
+            <div className="co-image-name-results__details">
+              {runsAsRoot(values.isi.image) && (
+                <Alert isInline className="co-alert" variant="warning" title="Image runs as root">
+                  Image <strong>{values.isi.name}</strong> runs as the <strong>root</strong> user
+                  which might not be permitted by your cluster administrator.
+                </Alert>
+              )}
+              <div className="row">
+                <div className="col-sm-3 col-md-2 hidden-xs text-right h2">
+                  <span
+                    className="fa fa-cube text-muted"
+                    style={{ fontSize: '100px' }}
+                    aria-hidden="true"
+                  />
+                </div>
+                <div className="col-sm-9 col-md-10">
+                  <h2 className="co-image-name-results__heading co-break-word">
+                    {values.isi.name}
+                    <small>
+                      {_.get(values.isi, 'result.ref.registry') && (
+                        <span>from {values.isi.result.ref.registry}, </span>
+                      )}
+                      <React.Fragment>
```

Comment:
Remove this fragment.

Reply:
ok

Reply:
I can still keep the `timestamp` inside it right?

Reply:
Yes.
You've been creating fragments and divs that are unnecessary containers for nodes that can stand alone.

Reply:
technically, I'm copying code and changing as little as possible